src/routes/index.js

```diff
@@ -123,9 +124,22 @@ module.exports = (app) => {
 
 		try {
 			let controller
-
 			if (req.params.file) {
-				controller = require(`@controllers/${version}/${controllerName}/${file}`)
+				let folderExists = fs.existsSync(
```

Comment:
@ankit1shahu why these changes add any issue you found on the mentoring routes file ?